internal/github/github.go

```diff
@@ -63,6 +63,18 @@ func (g *GHClient) CreateComment(org, repo string, number int, comment string) {
 	}
 }
 
+// CreateLabel creates a GitHub label to a specific repository if it doesn't exist.
+func (g *GHClient) CreateLabel(org, repo string, label github.Label) error {
+	g.logger.WithField("labels", label).Debug("Creating GitHub label")
+	_, _, err := g.GitHubClient.Issues.CreateLabel(context.Background(), org, repo, &label)
+	if err != nil {
+		g.logger.WithError(err).Debug("Failed to create GitHub label")
+		return err
```

Comment:
better to use `errors.Wrap(err, ".....")`

Reply:
this means to import a different library `"github.com/pkg/errors"` instead of `"errors"`.  Are there any objections to this @cpanato ?

Reply:
no that's the best practice @stafot . we do the same in couple of repos and just a bit context here https://dave.cheney.net/2016/06/12/stack-traces-and-the-errors-package

Reply:
Yes I saw that we use this one on cloud provisioner, thus I asked.

Reply:
import the library you need to, i prefer as well to wrap the error when you need to give some extra context